ios/RNMagicScriptTests/sources/specs/UiNodes/UiTextNodeSpec.swift

```diff
@@ -246,8 +246,8 @@ class UiTextNodeSpec: QuickSpec {
 
             context("when wrap disabled") {
                 let referenceBoundsSize = CGSize.zero
-                let shortTextRefereneceSizeForBounds = CGSize(width: 0.0426, height: 0.0144)
-                let veryLongTextRefereneceSizeForBounds = CGSize(width: 0.4986, height: 0.0144)
+                let shortTextRefereneceSizeForBounds: CGSize = is_iOS14() ? CGSize(width: 0.0444, height: 0.0168) : CGSize(width: 0.0426, height: 0.0144)
```

Comment:
is this only because we're using a different font for iOS 14?

Reply:
Yes, do you see any better way of testing font/text size?

Reply:
No, this is fine for now and it being linked to the same commit where we change the iOS 14 Font then it makes sense